.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,74 @@
+name: Build and Test
+on: [push]
+jobs:
+  tests:
+    name: Test
+    runs-on: ubuntu-20.04
+    container: ruby:2.6.6-slim-stretch
+    env:
+      BUNDLE_ENTERPRISE__CONTRIBSYS__COM: ${{ secrets.BUNDLE_ENTERPRISE__CONTRIBSYS__COM }}
+      CI: true
+      DATABASE_URL: "postgis://vets-api:postgres@127.0.0.1:5432/vets_api_test?pool=4"
+      DISABLE_BOOTSNAP: true
+      RAILS_ENV: test
+      "Settings.test_database_url": "postgis://vets-api:postgres@127.0.0.1:5432/vets_api_test"
+      "Settings.saml.cert_path": "spec/support/certificates/ruby-saml.crt"
+      "Settings.saml.key_path": "spec/support/certificates/ruby-saml.key"
+      "Settings.binaries.clamdscan": "clamscan"
+      TERM: xterm-256color
+      GIT_COMMIT_SHA: ${{ github.sha }}
+      CC_TEST_REPORTER_ID: ${{secrets.CC_TEST_REPORTER_ID}}
+      ImageOS: ubuntu20
```

Comment:
How does this work with a Debian image as specified on line 7?

Reply:
yeah it doesn’t 😄. need to change it